notebooks/ImvizExample.ipynb

```diff
@@ -17,27 +19,48 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "a4b3aee3",
    "metadata": {},
    "outputs": [],
    "source": [
     "import warnings\n",
     "warnings.simplefilter('ignore')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "dc45bbab-ddb7-4d24-8066-998a1ba17fd3",
+   "metadata": {},
+   "source": [
+    "We also need this to display Matplotlib in the notebook later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "53fb47da-0427-4a43-9d11-39567a12cdb0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "bccca7f4",
    "metadata": {},
    "outputs": [],
    "source": [
     "from astropy.utils.data import download_file\n",
     "\n",
-    "gc_2mass_j = download_file('https://www.astropy.org/astropy-data/galactic_center/gc_2mass_j.fits', cache=True)\n",
-    "gc_msx_e = download_file('https://www.astropy.org/astropy-data/galactic_center/gc_msx_e.fits', cache=True)"
+    "acs_47tuc_1 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03gjq_flc.fits', cache=True)\n",
```

Comment:
For my sanity, I am replacing the example images with real HST/ACS data. Everyone loves 47 Tuc, right? Or is it just calibration team? 😉